src/content/blog/using-frankenphp-with-ddev.md

```diff
@@ -0,0 +1,134 @@
+- Enabling/disabling Xdebug requires container rebuild
+
+If you want to suggest some feature or found a bug, feel free to [open an issue](https://github.com/stasadev/ddev-frankenphp/issues).
+
+## Running FrankenPHP in the Web Container
+
+Alternatively, FrankenPHP can be run inside the `web` container. This quickstart shows an example (for a Drupal 11 project) where FrankenPHP is added as an extra daemon.
+
+### ⚙️ Installation:
+
+```bash
+export FRANKENPHP_SITENAME=my-frankenphp-site
+mkdir ${FRANKENPHP_SITENAME} && cd ${FRANKENPHP_SITENAME}
+ddev config --project-type=drupal11 --webserver-type=generic --docroot=web --php-version=8.4
+ddev start
+
+cat <<'EOF' > .ddev/config.frankenphp.yaml
+web_extra_daemons:
+  - name: "frankenphp"
+    command: "frankenphp php-server --listen=0.0.0.0:80 --root=\"/var/www/html/${DDEV_DOCROOT:-}\" -v -a"
+    directory: /var/www/html
+web_extra_exposed_ports:
+  - name: "frankenphp"
+    container_port: 80
+    http_port: 80
+    https_port: 443
+EOF
+
+cat <<'DOCKERFILEEND' >.ddev/web-build/Dockerfile.frankenphp
+RUN curl -s https://frankenphp.dev/install.sh | sh
+RUN mv frankenphp /usr/local/bin/
+RUN mkdir -p /usr/local/etc && ln -s /etc/php/${DDEV_PHP_VERSION}/fpm /usr/local/etc/php
+DOCKERFILEEND
+
+ddev composer create-project drupal/recommended-project
+ddev composer require drush/drush
+ddev restart
+ddev drush site:install demo_umami --account-name=admin --account-pass=admin -y
+ddev launch
+# or automatically log in with
+ddev launch $(ddev drush uli)
+```
+
+### ⚠️ Limitations:
+
+- Can't install more PHP extensions (requires [ZTS build](https://github.com/oerdnj/deb.sury.org/issues/2208))
+- Limited debugging capabilities, `ddev xdebug` doesn't work
+
+## Benchmarking
+
+Using https://github.com/stasadev/ddev-frankenphp-benchmark, I compared `nginx-fpm` with `frankenphp` in the web container (`generic-web`) and `frankenphp` in the `frankenphp` container (`generic-addon`).
+
+<b>Software:</b><br>
+DDEV: v1.24.6<br>
+Mutagen: disabled<br>
+PHP: v8.4<br>
+Laravel: v12.19.3<br>
+FrankenPHP: v1.7.0<br>
+Docker Engine: v28.3.0<br>
+Operating System: Manjaro Linux AMD64<br>
+Kernel Version: 6.12.35-1-MANJARO
+
+<b>Hardware:</b><br>
+Intel i7 8750H (6 Core/12 Thread, 2.2 Ghz, Turbo 4.1 Ghz)<br>
+32 GB DDR4 2667 Mhz<br>
+Samsung 870 Evo SSD (530w/560r MB/s)
+
+![Requests](/img/blog/2025/07/frankenphp-requests.png)
+
+![Request Per Second](/img/blog/2025/07/frankenphp-request-per-second.png)
+
+![Transfer Per Second](/img/blog/2025/07/frankenphp-transfer-per-second.png)
+
+![Latency Distribution (Health Check Endpoint)](/img/blog/2025/07/frankenphp-latency-distribution-health-check.png)
+
+![Latency Distribution (Static Endpoint)](/img/blog/2025/07/frankenphp-latency-distribution-static.png)
+
+![Latency Distribution (HTTP Request Endpoint)](/img/blog/2025/07/frankenphp-latency-distribution-http-request.png)
+
+As you can see, there's no significant difference, but:
```

Comment:
Add a little more to the wrap-up. 

* Performance was adequate and similar
* However, support for FrankenPHP is a win where there is an upstream hosting environment using FrankenPHP.

Reply:
Added and moved the results up.